Elizabeth_Theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<@liferay_util["include"] page=top_head_include />
+
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+<#--  POSITION OF HEADER FIXED HERE!!!!!!!!  -->
+<header class="${header_css_class}" style="position:fixed">
```

Comment:
Do we really need an inline style or can we just add the necessary CSS to the CSS class?